plumex/post/edge_figs.py

```diff
@@ -0,0 +1,334 @@
+    "low 868": {"default": ("ea68e7", "low-868", "e90c88")},
+    "low 869": {"default": ("7314c2", "low-869", "db4a6c")},
+    "low 913": {"default": ("bc1c70", "low-913", "21a901")},
+    "med 914": {"default": ("0d3391", "med-914", "8dd223")},
+    "med 916": {"default": ("c41675", "med-916", "776714")},
+    "med 917": {"default": ("9be0a9", "med-917", "e8e683")},
+    "med 871": {"default": ("a42cc9", "med-871", "7cab4b")},
+    "hi 919": {"default": ("1e0610", "hi-919", "2557c9")},
+    "hi 920": {"default": ("c0ab39", "hi-920", "485d19")},
+    "no wind": {"default": ("db4841", "blender-nowind", "d7598b")},
+    "wind": {"default": ("61645e", "blender-wind", "a559a7")},
+    "low 1": {"default": ("677028", "low-1", "ea256d")},
+    "hi 1": {"default": ("729d03", "hi-1", "487513")},
+    "hi 2": {"default": ("56a332", "hi-2", "3115a5")},
+}
+
+
+vids = [
+    "low 862",
+    "low 865",
+    "low 867",
+    "low 869",
+    "low 913",
+    "med 871",
+    "med 914",
+    "med 916",
+    "hi 919",
+    "hi 920",
+]
+
+
+def _unpack_data(
+    center_regress_hash: str,
+    video_lookup_keyword: str,
+    edge_regress_hash: str,
+) -> RegressionData:
+
+    video, orig_center_fc = _load_video(data_lookup["filename"][video_lookup_keyword])
+    center_mitosis_step = mitosis.load_trial_data(
+        hexstr=center_regress_hash, trials_folder=trials_folder / "center-regress"
+    )
+    edge_mitosis_step = mitosis.load_trial_data(
+        hexstr=edge_regress_hash, trials_folder=trials_folder / "edge-regress"
+    )
+
+    center_fit_method = center_mitosis_step[1]["main"]
+    center_coeff_dc = center_mitosis_step[1]["regressions"][center_fit_method]["data"]
+
+    edge_plume_points = edge_mitosis_step[0]["data"]
+
+    center = cast(List[tuple[int, PlumePoints]], edge_plume_points["center"])
+    bot = cast(List[tuple[int, PlumePoints]], edge_plume_points["bottom"])
+    top = cast(List[tuple[int, PlumePoints]], edge_plume_points["top"])
+
+    top_edge_method, bot_edge_method = edge_mitosis_step[1]["main"]
+
+    edge_coefs_top = np.nanmean(
+        edge_mitosis_step[1]["accs"]["top"][top_edge_method]["coeffs"], axis=0
+    )
+    edge_coefs_bot = np.nanmean(
+        edge_mitosis_step[1]["accs"]["bot"][bot_edge_method]["coeffs"], axis=0
+    )
+
+    top_func = create_edge_func(edge_coefs_top, top_edge_method)
+    bot_func = create_edge_func(edge_coefs_bot, bot_edge_method)
+
+    start_frame = center[0][0]
+    end_frame = center[-1][0]
+
+    return RegressionData(
+        video=video[start_frame : end_frame + 1],
+        center_coef=center_coeff_dc,
+        center_func_method=center_fit_method,
+        center_plume_points=center,
+        top_plume_points=top,
+        bottom_plume_points=bot,
+        top_edge_func=top_func,
+        bot_edge_func=bot_func,
+        start_frame=start_frame,
+        orig_center_fc=orig_center_fc,
+    )
+
+
+vid_names = ["low 869", "med 914", "hi 920"]
+frame_ids = [250, 750, 1000, 1200]
```

Comment:
IIUC, these are for visualizing the fits, as opposed to the accuracy plots?  FWIW, it might help to have this up at the top of the file, so that if we have to tweak what experiment gets plotted, we don't have to go looking.